doc/classes/DisplayServer.xml

```diff
@@ -2833,19 +2833,19 @@
 			Scroll backward action, callback argument is not set.
 		</constant>
 		<constant name="ACTION_SCROLL_DOWN" value="12" enum="AccessibilityAction">
-			Scroll down action, callback argument is not set.
+			Scroll down action, callback argument is set to [code]0[/code] if scroll amount is line/item, and [code]1[/code] of scroll amount is page.
```

Comment:
Should the callback argument be constants themselves? Like `ACTION_SCROLL_LINE`, `ACTION_SCROLL_ITEM`, and `ACTION_SCROLL_PAGE`?

Reply:
Added enum constants for this and scroll_into_view hints.